dags/shared_tasks/mapping_tasks.py

```diff
@@ -266,7 +265,7 @@ def validate_endpoint_task(url, mapped_versions, params=None, **context):
         print(f"please validate manually: {list(errored_collections.keys())}")
         print("*" * 60)
 
-    if not len(errored_collections) == len(validations):
+    if len(errored_collections) == len(validations):
```

Comment:
@amywieliczka It seems like this was an error in logic, but I might not be understanding what exactly this is trying to achieve?